### PR TITLE
Zmiy Vladislav (task1 and task2)

### PR DESCRIPTION
**Pull request by Vlad0902**
Task 1

### CODE REVIEW

**Review by Dmitriy-NK — COMMENTED** [review]
File structure is OK. No additional change is required.

**Review by Dmitriy-NK — COMMENTED** [review]
1. График 4 (в большом наборе графиков) построен с нормированием к полному числу пассажиров. Предполагается сначала разбиение на подмножества а потом для каждого подмножества построение графика. у seaborn.kdeplot для этого используется аргумент "common_norm". Также цвета кривых при использовании hue задаются через аргумент "palette"
2. В разделе "graph down by gender" отсутствует график пропорционального распределения "# Next adjust graph to display the proportions of survival by gender"
3. И просьба добавить несколько предложений по своей оценке после "Add your summary about recieved data for raw value counts and proportionally" в том же разделе
4. К следующему разделу (Pclass) нет деления на выживших и не выживших, и добавить соответствующую оценку в вывод. [из мелкого предпочтительней использовать != 3 а не <=2 или <3]
5.  Ко всем моделям: приведите confusion matrix  к заданному в описании формату(включая нормализацию)
>При построении ROC кривой предпочтительно делать график квадратным и содержащим прямую (0,0) (1,1)

**Review by Dmitriy-NK — COMMENTED** [review]
Пункт 2 остался, отсутствует график нормированного(в долях единицы {не от общего количества а по подмножествам}) распределения выживших в двух подмножествах 'female','male'.
по 5 пункту, объяснял на лекции: sklearn.metrics.confusion_matrix считает метку 0 - положительной, а собственный порядок можно задать в аргументе labels [detailes](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)

**Review by Dmitriy-NK — DISMISSED** [review]
LGTM

**Review by Dmitriy-NK — APPROVED** [review]
Task 2 approved.
